frontend/src/pages/Chat.jsx

```
@@ -0,0 +1,415 @@
+                  className="relative group"
+                  ref={openMenuConversationId === conv.id ? menuRef : null}
+                >
+                  <div
+                    className={`
+                      p-3 rounded-lg cursor-pointer transition-all duration-200 flex items-center
+                      ${selectedConversation?.id === conv.id 
+                        ? "bg-gradient-to-r from-indigo-500 to-violet-600 text-white shadow-md" 
+                        : "bg-white text-gray-800 hover:bg-gray-50"}
+                    `}
+                    onClick={() => setSelectedConversation(conv)}
+                  >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)